truffle/src/main/ruby/core/rubinius/common/random.rb

```diff
@@ -0,0 +1,177 @@
+#
+# Extracted from https://github.com/rubinius/rubinius/blob/v2.4.1/kernel/common/random.rb
+#
```

Comment:
This needs to be the full Rubinius license. Can you copy from another file in `common`?